tests/test_google_sheets_pagination.py

```diff
@@ -61,6 +61,43 @@ def test_run(self):
                 # verify that we can paginate with all fields selected
                 self.assertGreater(record_count_by_stream.get(stream, 0), self.API_LIMIT)
 
+                record_count_sync = record_count_by_stream.get(stream, 0)
```

Comment:
There should not have needed to be any test changes here besides adding the failing sheet back to the test. I don't think these test additions are adding any test coverage. The sheets have been setup in a way that relies on a specific column with incrementing values used to compare against the sdc column. See fake_pk_list above.

Reply:
Removed extra assertion. The current test is written in such a way that it was respecting only `Pagination` stream. 
https://github.com/singer-io/tap-google-sheets/blob/25136fce609ca297256d63c2da2112fe2512ebae/tests/test_google_sheets_pagination.py#L80-L87
If we add back `sadsheet-pagination` then we have to write assertion according to this sheet also.

Reply:
That comment is misleading, the test is setup to apply to both sheets
https://github.com/singer-io/tap-google-sheets/blob/25136fce609ca297256d63c2da2112fe2512ebae/tests/test_google_sheets_pagination.py#L38
Additionally there is a large comment at the top of this test that should be removed https://github.com/singer-io/tap-google-sheets/blob/25136fce609ca297256d63c2da2112fe2512ebae/tests/test_google_sheets_pagination.py#L9-L13

Reply:
In the `sadsheet-pagination` sheet, there is a total of 238 rows and in that sheet row, no 199 and 200 are empty rows whereas in the `Pagination` sheet there are total of 239 rows with no empty row.
So, as two rows are empty in `sadsheet-pagination`, we need to write the separate assertion in which we are excluding rows no 199 and 200.